apps/backend/src/mutations/QuestionaryMutations.ts

```diff
@@ -147,6 +149,15 @@ export default class QuestionaryMutations {
           answer.questionId,
           answer.value
         );
+
+        const updatedAnswer = await this.dataSource.getAnswer(

```

Comment:
What if we do bulk `select` instead of single query? - This way. only one query calls, instead of n

Infact, we can even get the answer details from the `updateAnswer` by using `returning` in the postgres query. This way, we don't need to make a query to fetch at all.